cnd/src/btsieve/ethereum.rs

```diff
@@ -179,31 +222,36 @@ where
                 });
                 if !maybe_contains_transaction {
                     tracing::trace!(
-                        "bloom filter indicates that block does not contain transaction:
-                {:x}",
-                        block_hash,
+                        "bloom filter indicates that this block might contain an instance of the event"
```

Comment:
Ahm... is this correct? 😅 
I think this should be 
```suggestion
                        "bloom filter indicates that this block will not contain an instance of the event"
```

and in the else branch it should be 
`bloom filter indicates that this block might contain an instance of the event`